Common/src/grid_movement/CSurfaceMovement.cpp

```diff
@@ -3256,7 +3256,7 @@ void CSurfaceMovement::AeroelasticDeform(CGeometry *geometry, CConfig *config, u
   if (config->GetKind_GridMovement() == AEROELASTIC_RIGID_MOTION) {
     su2double Omega, dt, psi;
     dt = config->GetDelta_UnstTimeND();
-    Omega  = (config->GetRotation_Rate(3)/config->GetOmega_Ref());
+    Omega = config->GetRotation_Rate(2)/config->GetOmega_Ref();
```

Comment:
Using static types lets the compiler find bugs for us at compile time :)
Using bare-pointer does not let the compiler do anything :(

Reply:
I am curious how the compiler found that now that the underlying `Rotation_Rate` is static because is has 3 elements initialized to 0. So how did the compiler know but I guess I miss sth here

Reply:
Oh man am I blind... *insert palmface here*

Reply:
As blind an any programmer, we should have XYZ enums